src/extension_commands.mli

```diff
@@ -24,6 +24,10 @@ val open_terminal : command
 
 val switch_impl_intf : command
 
+module Dune_commands : sig
+  val open_current_dune_file : command
+end
+
```

Comment:
Having a separate module is unnecessary now. This can be flattened so `open_current_dune_file` is on the same level as the other commands.

Reply:
I want the `Dune` and `Dune_commands` modules to separate somehow dune-related constants and commands.

Reply:
I see your point, but can't we introduce separation later once it's necessary? A module with only one value is just adding noise at this point.